romana-install/roles/romana/postinstall/kubernetes/agent/tasks/romana_kube_agent.yml

```diff
@@ -5,7 +5,7 @@
     creates: /usr/local/lib/romana-kube-agent
   
 - name: Configure IP address for agent
-  replace: dest="/usr/local/lib/romana-kube-agent" regexp="192\.168\.0\.10" replace="{{ stack_nodes.Controller.mgmt_ip }}"
+  replace: dest="/usr/local/lib/romana-kube-agent" regexp="192\.168\.0\.10" replace="{{ romana_master_ip }}"
```

Comment:
Might it be possible to have the romana-kube-agent file as a template, where the IP address is correctly set to {{ romana_master_ip }}, instead of patching it up after the fact?